pkgs/development/python-modules/hetzner/default.nix

```
@@ -1,18 +1,26 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-}:
-
-buildPythonPackage rec {
+, head ? false
+}: let
+  version-info = if head then {
+    version = "0.9pre20221026";
+    rev = "e9f802c7b37a23efa3e6411364a5333c9c956258";
+    sha256 = "sha256-s0d0rTK5sZVE6H/cESvc/IXenDdM2oZvdlx6k5fA5W8=";
```

Prefer hash instead of sha256 
```suggestion
    hash= "sha256-s0d0rTK5sZVE6H/cESvc/IXenDdM2oZvdlx6k5fA5W8=";
```